crates/re_viewport/src/context_menu/mod.rs

```diff
@@ -183,6 +189,34 @@ struct ContextMenuContext<'a> {
     clicked_item: &'a Item,
 }
 
+impl<'a> ContextMenuContext<'a> {
+    /// Return the clicked item's parent container id and position within it.
+    ///
+    /// Valid only for space views, containers, and data results.
+    pub fn clicked_item_parent_id_and_position(&self) -> Option<(ContainerId, usize)> {
```

Comment:
the naming is a bit odd sicne you wouldn't get the parent when an entity is selected.
Maybe it's `clicked_item_parent_container_id_and_position`?
This is another case where a name for "container or space view" would be very useful.
Even so that's a not entirely accurate still since then I'd expect to get a parent container for a DataResult as documented

Reply:
Changed to slightly more explicit and less wrong `clicked_item_enclosing_container_id_and_position`